homeassistant/components/accuweather/__init__.py

```diff
@@ -87,11 +99,11 @@ def __init__(self, hass, session, api_key, location_key, forecast: bool):
         update_interval = timedelta(minutes=40)
         if self.forecast:
             update_interval *= 2
-        _LOGGER.debug("Data will be update every %s", update_interval)
+        _LOGGER.debug("Data will be update every %s", str(update_interval))
```

Comment:
It's not necessary to copy to string before logging the variable.

Reply:
It's not `str`, it's `timedelta` 🙂

Reply:
Yes, and we don't need to copy to string before logging a timedelta variable.
```
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from datetime import timedelta

In [2]: update_interval = timedelta(minutes=40)

In [3]: print(update_interval)
0:40:00

In [4]: print(str(update_interval))
0:40:00
```

Reply:
Actually, you're right. I thought it's needed for typing but mypy doesn't complain about it.

Reply:
Yeah, it's basically string formatting.

Reply:
I'll open PR to fix this.